pkg/alerting/condition/condition.go

```diff
@@ -95,7 +95,7 @@ func ParseAlertManagerWebhookPayload(annotations []gjson.Result) ([]*alertingv1.
 					errors = append(errors, fmt.Errorf("unhandled opni identifier %s", identifier))
 					opniRequests = append(opniRequests, nil)
 					anyFailed = true
-					break
```

Comment:
This logic isn't the same, was there originally a bug here? @alexandreLamarre

Reply:
Yeah I spoke to Alex about that.  The previous logic was incorrect and adding the label should result in the desired behaviour.